build.gradle

```diff
@@ -11,11 +11,11 @@ repositories {
 
 dependencies {
     compile group: 'javax.activation', name: 'activation', version: '1.1'
-    compile group: 'axis', name: 'axis', version: '1.2'
+    compile group: 'axis', name: 'axis', version: '1.2.1'
     compile group: 'axis', name: 'axis-saaj', version: '1.2'
     compile group: 'axis', name: 'axis-jaxrpc', version: '1.2'
     compile group: 'axis', name: 'axis-ant', version: '1.2'
-    compile group: 'commons-fileupload', name: 'commons-fileupload', version: '1.2.1'
+    compile group: 'commons-fileupload', name: 'commons-fileupload', version: '1.2.2'
```

Comment:
[Nexus Lifecycle](https://www.sonatype.com/product-nexus-lifecycle) has found that this version of `commons-fileupload` violates your company's policies.

Lifecycle recommends using version [1.4](https://search.maven.org/artifact/commons-fileupload/commons-fileupload/1.4/jar) instead as it does not violate any policies.